modules/nf-core/igv/js/main.nf

```diff
@@ -0,0 +1,66 @@
+
+process IGV_JS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::sed=4.7"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(alignment), path(index)
+
+    output:
+    tuple val(meta), path("*_genome-browser.html") , emit: browser
+    tuple val(meta), path(alignment)               , emit: align_files
+    tuple val(meta), path(index)                   , emit: index_files
```

Comment:
It won't emit these right? Because they are the same as the input channel?

Reply:
They are needed in the same location as the html. For example, for tests I copied them to the publishDir, so I can serve that directory and visualise the browser.

Reply:
Oh that's smart! Good idea!